src/client/gui/lib/settings/usage_settings.dart

```diff
@@ -40,10 +42,18 @@ class UsageSettings extends ConsumerWidget {
         },
       ),
       const SizedBox(height: 20),
+      HotkeyField(
+        value: hotkey,
+        onSave: (newHotkey) => ref.read(hotkeyProvider.notifier).set(newHotkey),
+      ),
+      const SizedBox(height: 20),
       PassphraseField(
         hasPassphrase: hasPassphrase,
         onSave: (value) {
-          ref.read(passphraseProvider.notifier).set(value);
+          ref
+              .read(passphraseProvider.notifier)
+              .set(value)
+              .onError(ref.notifyError((e) => 'Failed to set passphrase: $e'));
```

Comment:
I can't see this happening in the primary instance name case. Indeed it does happen in the passphrase field, which must be cleared after setting the passphrase, since it's not some data we want to keep around.